_benchmark/user-guide/concepts/time-latency.md

```diff
@@ -44,21 +33,35 @@ While the definition for _throughput_ remains consistent with other client-serve
 
 For more information about service time and latency in OpenSearch Benchmark, see the [Service time](#service-time) and [Latency](#latency) sections.
 
+## Took time
 
-### Service time
+**Took time** measures the amount of time the cluster spends processing a request on the server-side. The does not include the time it took for the request to reach the cluster from the client, or the time it took for the response to get back from the client to the cluster.
 
-OpenSearch Benchmark does not have insight into how long OpenSearch takes to process a request, apart from extracting the `took` time for the request. In OpenSearch, **service time** tracks the amount of time between when OpenSearch issues a request and receives a response.
+## Service time
+
+OpenSearch Benchmark does not have insight into how long OpenSearch takes to process a request, apart from extracting the [took time](#took-time) time for the request. In OpenSearch, **service time** tracks the amount of time between when OpenSearch issues a request and receives a response.
 
 OpenSearch Benchmark makes function calls to `opensearch-py` to communicate with an OpenSearch cluster. OpenSearch Benchmark tracks the amount of time between when the `opensearch-py` client sends a request and receives a response from the OpenSearch cluster and considers this to be the service time. Unlike the traditional definition of service time, the OpenSearch Benchmark definition of service time includes overhead, such as network latency, load balancer overhead, or deserialization/serialization. The following image highlights the differences between the traditional definition of service time and the OpenSearch Benchmark definition of service time.
 
 <img src="{{site.url}}{{site.baseurl}}/images/benchmark/service-time.png" alt="">
 
-### Latency
+## Latency
+
+There are two types of testing modes when using OpenSearch Benchmark, which affects the definition of latency:
+
+- [Benchmarking mode](#benchmarking-mode): Latency is measured the same as service time.
+- [Throughput-throttled mode](#throughput-throttled-mode): Latency is service-time plus the time a request spends waiting in the queue.
```

Comment:
I like how concise this section is. But I think we can reorder it to make it easier to understand. 
1. First sentence cam be rephrased to "There are two types of testing modes that can be activated by the `target-throughput` parameter in OpenSearch Benchmark. Both testing modes affect the definition of latency. 
2. Let's put the Benchmarking Mode section before the Throughput-Throttled Mode. 
3. We can encapsulate Benchmarking Mode and Throughput-Throttled Mode under a larger header called "Throughput"